docs/rules/prefer-destructuring-in-parameters.md

```diff
@@ -0,0 +1,38 @@
+# Prefer destructuring in parameters over accessing properties
+
+Makes your code shorter and nicer.
+
+This rule is fixable.
```

Comment:
There are some new requirements for rule docs, be sure to rebase on the latest `main` to pick them up.